e2e/tests/integration/staff-movements.ts

```diff
@@ -25,23 +27,36 @@ describe('Staff movements', () => {
   }
 
   describe('When adding staff movements on the desks and queues page', () => {
-    it("Should update the available staff when 1 staff member is added for 1 hour, and record the correct reason", () => {
-      cy
-        .asABorderForcePlanningOfficer()
-        .navigateHome()
-        .navigateToMenuItem('T1')
-        .selectCurrentTab()
-        .choose24Hours()
-        .openAdjustmentDialogueForHour('add', 0)
-        .adjustMinutes(60)
-        .adjustStaffBy(1)
-        .checkStaffMovementsOnDesksAndQueuesTabAre(1)
-        .checkStaffAvailableOnDesksAndQueuesTabAre(1)
-        .findAndClick('Staff Movements')
-        .checkStaffNumbersOnMovementsTabAre(1)
-        .checkUserNameOnMovementsTab(1, "Unknown")
-        .removeXMovements(1);
-    });
+    it("Should update the available staff when 1 staff member is added for 1 hour, and record the correct reason and " +
+      "it should appear in the export", () => {
+        const movementsCSV = "Terminal,Reason,Time,Staff Change,Made by" + "\n" +
+              "T1,Other start,"+ midnightThisMorning().format("YYYY-MM-DD") + " 00:00,1,\"Unknown\"" + "\n" +
+              "T1,Other end,"+ midnightThisMorning().format("YYYY-MM-DD") + " 01:00,-1,\"Unknown\""
+        cy
+          .asABorderForcePlanningOfficer()
+          .navigateHome()
+          .navigateToMenuItem('T1')
+          .selectCurrentTab()
+          .choose24Hours()
+          .openAdjustmentDialogueForHour('add', 0)
+          .adjustMinutes(60)
+          .adjustStaffBy(1)
+          .checkStaffMovementsOnDesksAndQueuesTabAre(1)
+          .checkStaffAvailableOnDesksAndQueuesTabAre(1)
+          .findAndClick('Staff Movements')
+          .checkStaffNumbersOnMovementsTabAre(1)
+          .checkUserNameOnMovementsTab(1, "Unknown")
+          .request({
```

Comment:
It's possible to use the href from a link in the dom so we don't have to hard code. I'll try to find the example